qcg.sh

```diff
@@ -0,0 +1,34 @@
+package: qcg
+version: "v1.0.6"
+requires:
+  - node
+  - QualityControl
+build_requires:
+  - node
+---
+#!/bin/bash
+
+mkdir -p "$INSTALLROOT/bin"
+npm install @aliceo2/qc@$PKGVERSION --only=production --loglevel error --no-save --no-package-lock --prefix $INSTALLROOT
+rsync -a --ignore-existing "$INSTALLROOT/node_modules/@aliceo2/qc/config-default.js" $INSTALLROOT/config.js
```

Comment:
cp -f

Reply:
I thought you wanted to always overwrite config.js in the installation ROOT, but I now understand what you are doing. You want to allow that user modified configurations get overwritten, right? Then the `rsync` is fine, indeed, for the wanted purpose.

 Iam not sure about it being a good idea in general, as I said elsewhere, but in the end it's your call.